src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/AssemblyContigAlignmentSignatureClassifier.java

```diff
@@ -136,6 +136,7 @@ static boolean hasIncompletePictureDueToOverlappingRefOrderSwitch(final AlignedC
         if (referenceSpanOne.contains(referenceSpanTwo) || referenceSpanTwo.contains(referenceSpanOne))
             return true;
 
+        // TODO: 10/29/17 this obsoletes the inverted duplication call code we have now, but those could be used to figure out how to annotate which known ref regions are invert duplicated 
```

Comment:
This TODO comment is also not very clear to me. Can you explicitly write down what needs to be done?